source/background.js

```diff
@@ -4,3 +4,23 @@ import dynamicContentScripts from 'webext-dynamic-content-scripts';
 // GitHub Enterprise support
 dynamicContentScripts.addToFutureTabs();
 domainPermissionToggle.addContextMenu();
+
+// `background` fetch required to avoid avoid CORB introduced in Chrome 73 https://chromestatus.com/feature/5629709824032768
+chrome.runtime.onMessage.addListener((
+  {action, payload},
+  sender,
+  sendResponse
+) => {
+  if (action === 'fetch') {
+    const {name} = payload;
+
+    // @ char (scoped repositories) is not escaped, only / is...
+    const url = 'https://registry.npmjs.org/' + name.replace('/', '%2F');
+
+    fetch(url)
+      .then(response => response.json())
+      .then(json => sendResponse(json));
+
+    return true; // Required to signal intent to respond asynchronously
```

Comment:
Can you do one last test? Try changing this function to be `async`, drop the return and use `await` instead of `then`. The last line should be `sendResponse(json)`

The docs say that it should return true, but perhaps a truth value also works (async functions return a promise, which is truthy)

Reply:
You mean something like that? 
```
chrome.runtime.onMessage.addListener(async (
  {action, payload},
  sender,
  sendResponse
) => {
  if (action === 'fetch') {
    const {name} = payload;

    // @ char (scoped repositories) is not escaped, only / is...
    const url = 'https://registry.npmjs.org/' + name.replace('/', '%2F');

    const response = await fetch(url);
    const json = await response.json();
    console.log(json)

    sendResponse(json);
  }
});
```? 

Not working tho. Befare I tried to return `Promise` but as you're saying, in docs there is explicitely mentioned `true`.

Reply:
Sad. Thanks for checking!